### PR TITLE
github/labeller: add more match patterns

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -1,40 +1,128 @@
...
-  - src/timedate/*
+  - '**/*timedate*'
 timesync:
-  - src/timesync/*
+  - '**/*timesync*'
+tests:
+  - src/test/**/*
+  - src/fuzz/**/*
+  - test/**/*
+  - .github/workflows/*
```
Comment (dtardon) [inline .github/labeler.yml:113]: Maybe also `**/test-*`?

File: .github/labeler.yml [hunk 2]
```diff
@@ -1,40 +1,128 @@
...
+  - man/*udev*
+uki:
+  - '**/ukify*'
+units:
+  - units/**/*
+userdb:
+  - '**/*userdb*'
+util-lib:
+  - src/basic/**/*
+  - src/shared/**/*
```
Comment (dtardon) [inline .github/labeler.yml:130]: Also `src/fundamental/*`?

**Comment by medhefgo** [comment]
nothing for sd-boot? :crying_cat_face:

**Comment by keszybz** [comment]
I made the three suggested changes. Since the CI doesn't do anything here, I'll just merge this. In the worst case, the labelling will not work, and we'll notice.

**Pull request merged**